### DISCUSSION TITLE
Hplm comparing phases and effects

### DISCUSSION THREAD

**Discussion by nrblsffspc** [jazznbass/scan#83] [Q&A] [answered]
I have a quick question about the hplm analyses: 

1) If I run the following code hplm(dat) and there are multiple phases in the dataset (e.g. 4 phases including baseline), to which difference between the phases does the level or slope effect refer? Is the baseline phase always taken as the reference point? 

2) How can I change the code to get the level or slope effects for the difference between the phases? So the level and slope effect from phase A to B, from B to C and from C to D. 

Or can I just subtract the effect sizes from each other? So if the effect of A to B is 0.6 and the effect of A to D is 0.8, is the effect of A to D the difference of these?

**Comment by researchtool** [comment, marked as answer]
Hi nrblsffspc.
Thanks for your interest in scan.
You can use the `contrast` argument to specify that. It defaults to `first` (B->A, C->A etc.), but you can set it to `preceding` to always compare to the preceding phase (B->A, C->B etc.). 

It's something we were discussing before, and there might be updates in the future that could allow more specific phase contrasts.

Cheers,
timo

**Comment by nrblsffspc** [comment]
Thanks a lot! It worked and we could proceed!
